de.fu_berlin.inf.dpp.intellij/src/de/fu_berlin/inf/dpp/intellij/editor/LocalEditorHandler.java

```diff
@@ -82,9 +79,9 @@ public Editor openEditor(
             return null;
         }
 
-        SPath path = virtualFileConverter.convertToPath(virtualFile);
+        SPath path = VirtualFileConverter.convertToSPath(virtualFile);
 
-        if (path == null) {
+        if (path == null || !SessionUtils.isShared(path)) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this use of "SessionUtils"; it is deprecated. ([squid:CallToDeprecatedMethod](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3ACallToDeprecatedMethod))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=294&issues=AWcxYM7E0EcSutaxLwSZ&open=AWcxYM7E0EcSutaxLwSZ)

[ISSUE_KEY]: AWcxYM7E0EcSutaxLwSZ